src/app/containers/MostRead/index.jsx

```diff
@@ -1,28 +1,40 @@
 import React, { useContext } from 'react';
 import { RequestContext } from '#contexts/RequestContext';
 import { ServiceContext } from '#contexts/ServiceContext';
-import useToggle from '../Toggle/useToggle';
+// import useToggle from '../Toggle/useToggle';
 import Canonical from './Canonical';
+import Amp from './Amp';
 
 const getLocalMostReadEndpoint = (service, variant) => {
-  const localhostURL = 'http://localhost:7080';
-  const localServiceURL = `${localhostURL}/${service}`;
+  const localServiceURL = `/${service}`;
 
   return variant
     ? `${localServiceURL}/most_read/${variant}.json`
     : `${localServiceURL}/most_read.json`;
 };
 
 const MostReadContainer = () => {
-  const { variant } = useContext(RequestContext);
-  const { service } = useContext(ServiceContext);
+  const { variant, platform } = useContext(RequestContext);
+  const { service, script } = useContext(ServiceContext);
 
-  const { enabled } = useToggle('mostRead');
-  if (!enabled) {
-    return null;
-  }
+  // const { enabled } = useToggle('mostRead');
+  // if (!enabled) {
+  // return null;
+  // }
 
```

Comment:
Since the props are the same for both variants, you could simplify this file with
```const MostReadComponent = platform === 'amp' ? Amp : Canonical;```
Then have
```
return <MostReadComponent ....>;
```